tests/apollo/test_skvbc_byzantine_primary_preexecution.py

```diff
@@ -0,0 +1,396 @@
+# Concord
```

Comment:
I see a lot of shared code between test_skvbc_batch_preexecution.py, test_skvbc_byzantine_primary_preexecution.py and test_skvbc_preexecution.py.
We have to figure out a way to reuse the code better. One more test suite and it will be unmaintainable.

This is out of scope for this PR of course.

Reply:
This is in this scope. Thanks for the comment. I will actually remove all the testcases and keep only first two as discussed with @teoparvanov . This will simplify the PR, we will be able to test the feature going for 1.4. Reduce the time to complete the build. Later this feature can be used by other platforms of testing who are using skvbc_replica. They can ask for any strategy and add Byzantine threats and do more testing. This PR and its sister PR is to create a boilerplate to add this capability. Thanks @tdimitrov for raising this.